Lib9c/Action/MigrateFee.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Generic;
+using System.Linq;
+using Bencodex.Types;
+using Libplanet.Action;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+using Nekoyume.Model.State;
+using Nekoyume.Module;
+
+namespace Nekoyume.Action
+{
+    [ActionType(TypeIdentifier)]
+    public class MigrateFee : ActionBase
+    {
+        public const string TypeIdentifier = "migrate_fee";
+        // TODO Will change once target address is determined
+        public static readonly Address TargetAddress = new Address();
+
+        public List<Address> FeeAddresses;
```

Comment:
Can we not use pre-calculated fee addresses? When I looked at the `Execute` logic, it seems to be able to take any NCG, even if the address is not the fee address. (e.g. StakeState's NCG, Agent's NCG, etc)

If it is impossible, the current logic is also okay.